File: Modul 3/SOAL 5/PRAK205-2210817120003-IKHTIARA FAKHRUNISA.c
#include <stdio.h>
#include <math.h>

int main(){

    for(int i = 0; i <= 1; i++){
    float a, t, k, l, m;
    scanf("%f %f", &t, &m);

    a = sqrt(m*m-t*t);
    k = t + m + a;
    l = 0.5 * a * t;
    printf("Alas = %.f cm\n", a);
    printf("Tinggi = %.f cm\n", t);
    printf("Keliling = %.f cm\n", k);
    printf("Luas = %.f cm^2", l);
      }
    return 0;
}

File: Modul 3/SOAL 5/PRAK205-2210817120003-IKHTIARA FAKHRUNISA.py
import math
tinggi, miring = input().split()
tinggi = float(tinggi)
miring = float(miring)
alas = math.sqrt(miring * miring - tinggi * tinggi)
keliling = tinggi + miring + alas
luas = 0.5 * alas * tinggi
print("Alas =", round(alas), "cm")
print("Tinggi =", round(tinggi), "cm")
print("Keliling =", round(keliling), "cm")
print("Luas =", round(luas), "cm^2\n")

tinggiB = input()
miringB = input()
tinggiB = float(tinggiB)
miringB = float (miringB)
alasB = math.sqrt(miringB * miringB - tinggiB * tinggiB)
kelilingB = tinggiB + miringB + alasB
luasB = 0.5 * alasB * tinggiB
print("Alas =", round(alasB), "cm")
print("Tinggi =", round(tinggiB), "cm")
print("Keliling =", round(kelilingB), "cm")
print("Luas =", round(luasB), "cm^2")